src/bridge/IrcHandler.ts

```diff
@@ -148,19 +148,39 @@ export class IrcHandler {
      * @return {Promise} which is resolved when the PM room has been created.
      */
     private async createPmRoom (toUserId: string, fromUserId: string, fromUserNick: string, server: IrcServer) {
+        const users: {[userId: string]: number} = { };
+        users[toUserId] = 10;
+        users[fromUserId] = 100;
```

Comment:
10 and 100 are repeatedly used with the meaning of `lowPowerLevel` and `highPowerLevel` which I think should be constants to write more descriptively and remove potential errors when updating these numbers in the future.

Reply:
Mm, potentially. I think we use these powerlevels in multiple places so it might need a sensible refactor anyway.